### ISSUE TITLE
Final Report Review

### ISSUE THREAD

**Issue by AaronZang** [andrewkozma/4741-Project#20]
**Things I liked about the project:**
- The team tried several interesting models and analyzed why the models performed good / bad.
- I like the idea of Random Forest and the way you used Gini Index to identify which variable is the most important. Really interesting. And it turned out that Random Forest had the best performance among all the ML models.
- The team did a good job on finding the important factors that may affect player's shot - and the result is reasonable.
- The topic is clear and well-defined.
- The charts and graphs are well visualized.

**Things can be improved in the future:**
- I'm wondering how big the dataset is and how did you split the data into training, validation, and test set.
- The team tried several methods to avoid overfitting, but without knowing the detail about dataset, I'm wondering why the models tend to overfit rather than underfit.
- It would be really interesting to develop the model for each position since different players have their own "comfort zone" of shooting.

In all, congrats to the team on finishing this interesting project!
